wordfeudbot/main.py

```diff
@@ -663,327 +666,9 @@ def word_to_tile_position(move, tiles):
-
-                    # Check if it is reasonable to swap tiles
-                    if points < 20 and smart_points < 20 and len(vocals_on_hand) < 2 and len(consonants_on_hand) < current_game.tiles_in_bag:
-                        # Swap all consonants in order to get more vocals
-                        logging.info(
-                            f'Swapping {len(consonants_on_hand)} consonants in hand')
-                        wf.swap_tiles(current_game.game_id, consonants_on_hand)
-                        break
-                    elif points < 20 and smart_points < 20 and len(consonants_on_hand) < 2 and len(vocals_on_hand) < current_game.tiles_in_bag:
-                        # Swap all vocals in order to get more consonants
-                        logging.info(
-                            f'Swapping {len(vocals_on_hand)} vocals in hand')
-                        wf.swap_tiles(current_game.game_id, vocals_on_hand)
-                        break
-
-                    tile_positions = word_to_tile_position(
-                        (x, y, horizontal, word, points), current_game.tiles)
-
-                    try:
-                        # If move was accepted by the server
-                        wf.place_tiles(current_game, word, tile_positions)
-                        logging.info(f'Placed "{word}" for {points} points')
-                        if points > HIGH_POINTS_THRESHOLD:
-                            # Send response message to user
-                            wf.send_chat_message(
-                                current_game.game_id, random.choice(player_word_high_points_messages))
-                        break
-                    except AssertionError:
-                        # If move was invalid
-                        logging.warning(f"An invalid move was made: {word}")
-                else:
-                    # If no move was accepted by the server
-                    # (same result as if no move was found)
-                    if current_game.tiles_in_bag < 7:
-                        logging.warning("No moves available, skipping turn")
-                        wf.skip_turn(current_game.game_id)
-                    else:
-                        logging.warning(
-                            "No moves available, replacing all tiles")
-                        letter_list = current_game.letters
-                        wf.swap_tiles(current_game.game_id, letter_list)
-
-        # Update timestamp for next iteration
-        if random.randint(0, 1000):
-            last_check_unix_time = current_unix_time
-        else:
-            last_check_unix_time = 0
-
-        # Sleep between every iteration
-        time.sleep(PLAYING_SPEED)
-
-
-def is_emoji(input_string: str):
-    for character in input_string:
-        if not character in UNICODE_EMOJI:
-            return False
-    return True
-
-
-if __name__ == '__main__':
+def main():
```

Comment:
Codacy found an issue: [main is too complex (39) (MC0001)](https://app.codacy.com/manual/admin_136/wordfeud-bot/pullRequest?prid=5763881)